api/services/Search.js

```diff
@@ -28,7 +38,7 @@ module.exports = {
       }
 
       if (opts.visibility) {
-        qb.where('groups.visibility', opts.visibility)
+        qb.whereIn('groups.visibility', opts.visibility)
```

Comment:
does this work if opts.visibility is not an array? i want to make sure we dont break existing queries from mobile app

Reply:
There are three cases of this pattern, in forGroups, forTags and forPosts, and this was the only `where`. The other two are `whereIn`. @lorenjohnson do you recall if the mobile app has any group search queries, especially if they pass visibility? I figured we should standardize on array as input (although I think makeModels.js has a default value that I would need to change as well)